.github/workflows/nightly-build.yml

```diff
@@ -0,0 +1,41 @@
+name: Nightly build
+
+on:
+  schedule:
+    # only on weekdays
+    - cron:  '0 0 * * 2-6'
```

Comment:
Technically, this is Tue-Sat, and the reason for that is so we get the changes from Friday sooner rather than later (Sat vs Mon).

Reply:
@hariso I wondered about that when I noticed it in conduit's repo. I'll update the comment to make it clearer